contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -184,23 +214,34 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
         newTickUpper = nearestTickMultiple(int24(newTickUpperExact), tickSpacing);
 
         // Call to VoltzVault contract to update the position lower and upper ticks
-        _vault.rebalance(IVoltzVault.TickRange(newTickLower, newTickUpper));
+        vault.rebalance(IVoltzVault.TickRange(newTickLower, newTickUpper));
 
-        emit RebalancedTicks(newTickLower, newTickUpper);
+        emit RebalancedTicks(vault, newTickLower, newTickUpper);
         return (newTickLower, newTickUpper);
     }
 
-    /// @notice Callback function called after for ERC20RootVault::deposit
-    function depositCallback() external override {
+    function pushFunds() external {
+        _requireAtLeastOperator();
+        
+        // 0. Set the local variables
         address[] memory tokens = _tokens;
-        IERC20Vault erc20Vault = _erc20Vault;
+        uint256 balance = IERC20(tokens[0]).balanceOf(address(_erc20Vault));
```

Comment:
Shall we store _erc20Vault in local var and replace it here and in the for loop?